frontend/packages/dev-console/integration-tests/features/operators.feature

```diff
@@ -83,16 +83,28 @@ Scenario: Install OpenShift Virtualization Operator: VM-01-TC01
 
 
 @regression, @smoke
-Scenario: Create HyperConverged Cluster: VM-01-TC02
+Scenario: Create OpenShift Virtuliaztion Deployment
```

Comment:
Is this scenario not related to VM-01-TC02?

Reply:
updated